clients/client/test/helper.js

```diff
@@ -23,3 +26,17 @@ exports.withRestoredEnvVars = () => {
     }
   });
 };
+
+const ROOT_DIR = path.resolve(__dirname, '../../..');
+const suiteName = () => {
```

Comment:
this is copied from lib/testing to avoid having to implicitly depend on package from outside.